privacy_policy/en/privacy-notice-product-temp-mail.md

```diff
@@ -1,18 +1,30 @@
 # AdGuard Temp Mail Privacy Notice
 
```

Comment:
Date?

Reply:
In other privacy notices we place it in the beginning

Reply:
I suggest that we start placing it at the end of the document, what do you think?

Reply:
Let's do it later, for now let's keep this consistent with other notices